Add admin urls to docs - #11963 (#11965)
wagtail/wagtail: docs/extending/admin_views.md

This will result in a top-level menu item "Agenda" with the two viewsets' menu items as sub-items, e.g. "Calendar" and "Events".

For further customizations, refer to the {class}`~wagtail.admin.viewsets.base.ViewSetGroup` documentation.

## Adding links in admin views
### Snippets

> In python you can use `MySnippet.get_url_name(view_name)` to get the name of the snippet view.

Reviewer: ```suggestion
In Python, you can use `MySnippet.get_url_name(view_name)` to get the name of the snippet view.
```

> In a template the snippets url names by default are wagtailsnippets_{app_label}_{model_name}:{list/edit/inspect/copy/delete}

Reviewer: ```suggestion
In a template the snippets url names by default are `wagtailsnippets_{app_label}_{model_name}:{list/edit/inspect/copy/delete}`
```

> For example linking the BreadTypeSnippet in the bakery demo would look like this

Reviewer: ```suggestion
For example linking the `BreadTypeSnippet` in the bakery demo would look like this
```

> ```{code-block} html

Reviewer: ```suggestion
```html+django
```

{% url 'wagtailsnippets_breads_breadtype:list' %}
{% url 'wagtailsnippets_breads_breadtype:edit' object.id %}
{% url 'wagtailsnippets_breads_breadtype:inspect' object.id %}
{% url 'wagtailsnippets_breads_breadtype:copy' object.id %}
{% url 'wagtailsnippets_breads_breadtype:delete' object.id %}
```
### Pages

New page

> ```html

Reviewer: ```suggestion

```html+django
```

{% url 'wagtailadmin_pages:add' content_type_app_name content_type_model_name parent_id %}
```

Page usage

> ```html

Reviewer: ```suggestion

```html+django
```

{% url 'wagtailadmin_pages:usage' page_id %}
```

Edit page

> ```html

Reviewer: ```suggestion

```html+django
```

{% url 'wagtailadmin_pages:edit' page_id %}
```

Delete page

> ```html

Reviewer: ```suggestion
```html+django
```

{% url 'wagtailadmin_pages:delete' page_id %}
```

Copy page

> ```html

Reviewer: ```suggestion+django
```html
```

{% url 'wagtailadmin_pages:copy' page_id }
```

### Images

Images list

> ```html

Reviewer: ```suggestion
```html+django
```

{% url 'wagtailimages:index' %}
```

Edit image

> ```html

Reviewer: ```suggestion
```html+django
```

{% url 'wagtailimages:edit' image_id %}
```

Delete image

> ```html

Reviewer: ```suggestion
```html+django
```

{% url 'wagtailimages:delete' image_id %}
```

New image

> ```html

Reviewer: ```suggestion

```html+django
```

{% url 'wagtailimages:add' %}
```

Image usage

> ```html

Reviewer: ```suggestion
```html+django
```

{% url 'wagtailimages:image_usage' image_id %}
```

### AdminURLFinder
To find the url for any model in the admin the `AdminURLFinder` class can be used.

> ```{code-block} python

Reviewer: ```suggestion
```python
```

This will result in a top-level menu item "Agenda" with the two viewsets' menu items as sub-items, e.g. "Calendar" and "Events".

For further customizations, refer to the {class}`~wagtail.admin.viewsets.base.ViewSetGroup` documentation.

## Adding links in admin views

> ### Snippets
> In Python, you can use `MySnippet.get_url_name(view_name)` to get the name of the snippet view.
> In a template the snippets url names by default are `wagtailsnippets_{app_label}_{model_name}:{list/edit/inspect/copy/delete}`
> 
> For example linking the `BreadTypeSnippet` in the bakery demo would look like this

Reviewer: How about this?

```suggestion

### Snippets

We will use `BreadTypeSnippet` from the [Wagtail Bakery demo](https://github.com/wagtail/bakerydemo/) as an example.

In Python, you can use {meth}`~wagtail.admin.viewsets.base.ViewSet.get_url_name` to get the name of the snippet view URL. (e.g. `BreadTypeSnippet.get_url_name("list")`)

In templates, the snippet URL names follow the following pattern: `wagtailsnippets_{app_label}_{model_name}:{list/edit/inspect/copy/delete}`

So the `BreadTypeSnippet` URLs would look like:
```
